smoke-test-caldera.bash

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# check that the relay is working
+listen_to_sequencer_feed() {
+    #  Listen to the sequencer feed and check if the sender address is detected
+    while read -r message; do
+        # Check if the message contains the specific sender address
+        if [[ "$message" == *"\"sender\":\"0xdd6bd74674c356345db88c354491c7d3173c6806\""* ]]; then
+            echo "Sender address detected"
+            break
+        fi
+    done < <(wscat -c ws://127.0.0.1:9652)
```

Comment:
Just for my own understanding, this line is piping `wscat` into `stdin` for `read` at the beginning of the loop correct?

Reply:
yes